### PR TITLE
Update gobdokumente to 1.3.1

### PR DESCRIPTION
**Pull request by herrriehm**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` left no offenses.
- [x] The commit message includes the cask’s name and version.

added alternative name, depends_on macos

### CODE REVIEW

File: Casks/gobdokumente.rb [hunk 1]
```diff
@@ -6,8 +6,11 @@
   url 'https://download.moapp.software/GoBDokumente.zip'
   appcast 'https://sparkle.moapp.software/gobdokumente.xml'
   name 'GoBDokumente'
+  name 'GoBDocuments'
```
Comment (vitorgalvao) [inline Casks/gobdokumente.rb]: This is not a place for common misspellings. Is the app caslled this anywhere?

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

File: Casks/gobdokumente.rb (hunk 1 above)
Comment (herrriehm) [reply Casks/gobdokumente.rb]: Thanks for asking. „GoBDocuments“ is not a spelling mistake, but merely the official English name of the program. You can find (German) „GoBDokumente“ on the [German website](https://gobdokumente.de/deutsch.html), and (English) „GoBDocuments“ on the [English website](https://gobdokumente.de/index.html).
Although obviously the English website is preferred, I propose to stay with the German name „GoBDokumente“ as appcast and url also say so.

File: Casks/gobdokumente.rb [hunk 2]
```diff
@@ -6,8 +6,11 @@
...
   homepage 'https://gobdokumente.de/'
 
+  depends_on macos: '>= :el_capitan'
```
Comment (vitorgalvao) [inline Casks/gobdokumente.rb]: Always release names, never numbers.

Comment (vitorgalvao) [reply Casks/gobdokumente.rb]: Always release names, never numbers.

**Pull request merged**